LoopUI/Views/LoopCompletionHUDView.swift

```diff
@@ -150,11 +165,18 @@ public final class LoopCompletionHUDView: BaseHUDView {
                     caption?.text = timeString
                 }
 
-                accessibilityLabel = String(format: LocalizedString("Loop ran %@ ago", comment: "Accessbility format label describing the time interval since the last completion date. (1: The localized date components)"), timeString)
-
-                if let fullTimeStr = formatterFull.string(from: ago) {
-                    lastLoopMessage = String(format: LocalizedString("%1$@ last successfully completed a loop %2$@ ago.", comment: "Last loop time completed message (1: app name) (2: last loop time string)"), Bundle.main.bundleDisplayName, fullTimeStr)
+                var timeDateString: String = ""
+                if ago < timeAgoToIncludeDate {
+                    timeDateString = timeFormatter.string(from: date)
+                } else {
+                    timeDateString = timeDateFormatter.string(from: date)
                 }
+
+                let timeString = String(format: LocalizedString("%1$@ ago at \n%2$@", comment: "Format string describing the time interval since and time of the last completion date. (1: The localized date components, (2: the date"), formatter.string(from: ago)!, timeDateString)
+
+                accessibilityLabel = String(format: LocalizedString("Loop ran %@", comment: "Accessbility format label describing the time interval since and time of the last completion date. (1: last loop time string)"), timeString)
+
+                lastLoopMessage = String(format: LocalizedString("%1$@ last successful loop was\n%2$@", comment: "Last time loop completed message (1: app name) (2: last loop time string)"), Bundle.main.bundleDisplayName, timeString)
```

Comment:
"Loop last successful loop.." is odd grammar. Prefer previous wording here. "Loop last  successfully completed a Loop" or drop the app name: "Last successful loop was ..."

Reply:
Modified language